content/hi/docs/contribute/localization.md

```diff
@@ -0,0 +1,437 @@
+---
+title: कुबेरनेट्स प्रलेखन का स्थानीयकरण
+content_type: concept
+approvers:
+- divya-mohan0209
+- Garima-Negi
+weight: 50
+card:
+  name: contribute
+  weight: 50
+  title: डॉक्स का अनुवाद
+---
+
+<!-- overview -->
+
+यह पृष्ठ आपको दिखाता है कि कैसे विभिन्न
+[localize](https://blog.mozilla.org/l10n/2011/12/14/i18n-vs-l10n-whats-the-diff/) भाषाओं मे
+दस्तावेज़ (Docs) करे |
+
+
+<!-- body -->
+
+## मौजूदा स्थानीयकरण में योगदान करें
+
+आप मौजूदा स्थानीयकरण कंटेंट में लिखने या सुधारने में मदद कर सकते हैं। [कुबेरनेट्स
+स्लैक](https://slack.k8s.io/) में आपको प्रत्येक स्थानीयकरण के लिए एक चैनल मिलेगा| यहाँ एक सामान्य
+[SIG डॉक्स स्थानीयकरण स्लैक चैनल](https://kubernetes.slack.com/messages/sig-docs-localizations) भी है जहाँ आप हेलो कह सकते हैं।
+
+{{< note >}}
+यदि आप किसी ऐसे स्थानीयकरण पर काम करना चाहते हैं जो पहले से मौजूद है, तो जाँच करें
```

Comment:
This doesn't translate well. Please could you check and redo the machine translation so that it translates to what the original content tried to convey.